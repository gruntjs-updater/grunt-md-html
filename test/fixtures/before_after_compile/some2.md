some2.md
====

```javascript
var a = 10;
```