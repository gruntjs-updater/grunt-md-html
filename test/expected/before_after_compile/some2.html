<h1 id="some2-md">some2.md</h1>
<pre><code class="lang-javascript"><span class="hljs-variable"><span class="hljs-keyword">var</span> a</span> = <span class="hljs-number">10</span>;
</code></pre>
<h1 id="some3-md">some3.md</h1>
<p>Hello, I am some3.md</p>
<h2 id="hello-helinjiang">Hello grunt-md-html</h2>
